docs/reference/commandline/daemon.md

```diff
@@ -552,6 +553,25 @@ files. For example:
     --cluster-store-opt kv.keyfile=/path/to/key.pem
 ```
 
+The currently supported cluster store options are:
+
+*  `kv.cacertfile`
```

Comment:
Thanks @dhiltgen  Change all `will be used` to `is used` throughout all files.

![image](https://cloud.githubusercontent.com/assets/1347209/10417437/457480b0-6ff0-11e5-9dd0-e9f997bf505c.png)